src/find-options/FindOneOptions.ts

```diff
@@ -59,4 +59,9 @@ export interface FindOneOptions<Entity = any> {
      */
     loadEagerRelations?: boolean;
 
+    /**
+     * If this is set to true, SELECT query in a `find` method will be executed in a transaction.
+     */
+    transaction?: boolean
```

Comment:
I'm assuming this was intentionally added but seems to be a separate feature from the rest of this PR

Reply:
oh yeah I forgot I need to create a separate branch for this separate feature